noxfile.py

```diff
@@ -0,0 +1,18 @@
+import nox
+
+
+@nox.session(python=['3.7'])
```

Comment:
Nice work!  Do like the format of this file better.  I find it more readable and less magical for general Python developers who aren't experts in `tox`.